style.css

```diff
@@ -0,0 +1,68 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Arvo:wght@700&family=Cutive+Mono&family=Montserrat:ital,wght@0,100;1,900&family=Recursive:wght@300&family=Roboto+Mono:wght@100&family=Splash&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Arvo', serif;
+}
+body{
+    min-height: 100vh;
+    background-color: #bababa;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.paper{
+    position: relative;
```

Comment:
Group similar properties like this..in each selector.
    width: 90%;
    min-width: 400px;
    max-width: 800px;
    height: 480px;

    margin: 0 auto;

    background-color: #fafafa;
    border-radius: 10px;
    overflow: hidden;
}

Reply:
yes sure
for understanding purposes, I did not group these properties
sure I will, from next time...